src/components/ImageSlider/Arrow.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import { View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+class Arrow extends Component {
+  render() {
+    const {
+      name,
+      color,
+      enableBackground,
+      backgroundColor,
+      backgroundRounding,
+      enableBorder,
+      borderSize,
+      borderColor,
+      size,
+      onPress,
+      style,
+      disabled,
+    } = this.props
+
+    const viewHeight =
+      size + (enableBackground && enableBorder ? borderSize : 0)
+
+    const viewStyle = {
+      backgroundColor: enableBackground ? backgroundColor : null,
+      borderRadius: enableBackground ? backgroundRounding : null,
+      borderColor: enableBorder && enableBackground ? borderColor : null,
+      borderWidth: enableBorder && enableBackground ? borderSize : null,
+      height: viewHeight,
+      width: viewHeight,
+      display: 'flex',
+      alignSelf: 'center',
+      alignItems: 'center',
+      justifyContent: 'center',
+      position: 'absolute',
+      zIndex: 2,
+      opacity: disabled ? 0.5 : 1,
+    }
+    return (
```

Comment:
Why not use a `TouchableOpacity` around the view? Right now clicking on the background won't do anything. Ideally you'd have a hitslop around the actual visible background.